src/main/scala/com/linkedin/feathr/offline/job/FeatureJoinJob.scala

```diff
@@ -157,7 +157,12 @@ object FeatureJoinJob {
     val (joinedDF, _) = getFeathrClientAndJoinFeatures(ss, observationsDF, featureGroupings, joinConfig, jobContext, localTestConfig)
 
 
-    val parameters = Map(SparkIOUtils.OUTPUT_PARALLELISM -> jobContext.numParts.toString, SparkIOUtils.OVERWRITE_MODE -> "ALL")
+    var parameters = Map(SparkIOUtils.OUTPUT_PARALLELISM -> jobContext.numParts.toString, SparkIOUtils.OVERWRITE_MODE -> "ALL")
+
+    if (ss.conf.get("spark.feathr.outputFormat", "").nonEmpty) {
+      parameters = parameters + (SparkIOUtils.OUTPUT_FORMAT -> ss.conf.get("spark.feathr.outputFormat"))
```

Comment:
why not just output.format?

Reply:
guess I'm trying to be consistent with other code